zebra/zebra_rib.c

```diff
@@ -1212,6 +1209,12 @@ static int rib_can_delete_dest(rib_dest_t *dest)
 		return 0;
 	}
 
+	/*
+	 * Unresolved rnh's are stored on the default route's list
+	 */
+	if (is_default_prefix(&dest->rnode->p))
```

Comment:
NB: `dest->rnode` can be the source prefix node in an IPv6 sourcedest table.  Luckily, the prefix of a source prefix node can never be the default prefix.  But it might be good to point this out in a comment.

Reply:
comment added